src/dns/aardvark.rs

```diff
@@ -89,17 +103,37 @@ impl Aardvark {
         }
 
         log::debug!("Spawning aardvark server");
-        Command::new("systemd-run")
-            .args([
-                "-q",
+
+        if Aardvark::is_executable_in_path("systemd-run") {
```

Comment:
We could avoid this in favor of using systemd's dbus API - saves us a fork-exec and a lot of problems dealing with their CLI. I won't force you to do it here but we should do it before we release.

Reply:
Yes. I have added a comment `TODO` here. I'll check if we have api for this then i'll modify this.